### ISSUE TITLE
Docker installation not working 

### ISSUE THREAD

**Issue by muke5hy** [gramener/gramex#144]
I followed `https://learn.gramener.com/guide/install/#docker-install` for installation 

After running `docker run --name gramex-instance -p 9988:9988 gramener/gramex` 

I got the following error. 

```
DEBUG   03-Apr 12:56:15 config PORT Loading config: /opt/conda/lib/python3.6/site-packages/gramex/apps/ui/gramex.yaml
DEBUG   03-Apr 12:56:16 __init__ PORT Running callback: app

-------------------------------------------------------------------------
Read the license agreement at https://learn.gramener.com/guide/license/

Do you accept the license (Y/N): Traceback (most recent call last):
  File "/opt/conda/bin/gramex", line 11, in <module>
    sys.exit(commandline())
  File "/opt/conda/lib/python3.6/site-packages/gramex/__init__.py", line 177, in commandline
    callback(**kwargs)
  File "/opt/conda/lib/python3.6/site-packages/gramex/install.py", line 550, in run
    gramex.init(cmd=AttrDict(app=app_config['run']))
  File "/opt/conda/lib/python3.6/site-packages/gramex/__init__.py", line 294, in init
    callbacks[key]()
  File "/opt/conda/lib/python3.6/site-packages/gramex/services/__init__.py", line 153, in callback
    gramex.license.accept()
  File "/opt/conda/lib/python3.6/site-packages/gramex/license.py", line 39, in accept
    result = six.moves.input('Do you accept the license (Y/N): ').strip()
EOFError: EOF when reading a line
INFO    03-Apr 12:56:16 capturehandler PORT chromecapture.js terminated: pid=50
```

**Comment by sanand0** [comment]
Thanks @muke5hy -- I missed that. We don't need to accept the license in the open source version. I will put in a fix in a day or two.

**Comment by joshuamosesb** [comment]
It'll be much help to docker-beginners, to add the below command to the [docker-install docs-page](https://learn.gramener.com/guide/install/#docker-install)
```
docker run --network=host \
               --name gramex-agri-prod \
               -d -i -t \
               -p <port_given_in_gramex.yaml>:<port_given_in_gramex.yaml> \
               -v "$(pwd)":"$(pwd)" -w "$(pwd)" \
               gramener/gramex
```
- `--network=host`: enables access to host machine using localhost (esp for DB connection)
- `-d`: enables running gramex as a background process
- `-p <port_given_in_gramex.yaml>:<port_given_in_gramex.yaml>`: port for gramex to listen (might be redundant with `--network`
- `v "$(pwd)":"$(pwd)" -w "$(pwd)"`: for starting gramex from the current project directory

**Comment by sanand0** [comment]
We've fixed this with Gramex 1.63. Hope it works for everyone -- and sorry about the year long delay!
